### ISSUE TITLE
CLI bug: the argument '-j' cannot be used with '[INPUT]'

### ISSUE THREAD

**Issue by sanj0** [hjson/hjson-rust#35]
I have just installed the hjson CLI using `cargo install hjson` (version is 1.0.0). Upon trying to convert a HJSON-file to JSON using the example command from the README, I got the following error:

```
$ hjson -j books.hjson
error: the argument '-j' cannot be used with '[INPUT]'

Usage: hjson -j [INPUT]

For more information, try '--help'.
```

I assume this to be an issue in how clap is configured.

**Comment by trobro** [comment]
Thanks for reporting this issue. Fixed in release v1.1.0 now available on crates.io.

**Comment by speicherwerk** [comment]
Awesome, thank you very much!
